adapter.go

```diff
@@ -15,6 +15,7 @@
 package xormadapter
 
 import (
```

Comment:
README needs to be updated: https://github.com/casbin/xorm-adapter?tab=readme-ov-file#context-adapter